var/spack/repos/builtin/packages/fpocket/package.py

```diff
@@ -3,23 +3,35 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+
 from spack.package import *
 
 
 class Fpocket(MakefilePackage):
-    """fpocket is a very fast open source protein pocket detection algorithm
-    based on Voronoi tessellation."""
+    """The fpocket suite of programs is a very fast open source
+    protein pocket detection algorithm based on Voronoi tessellation."""
 
     homepage = "https://github.com/Discngine/fpocket"
-    version("master", branch="master", git="https://github.com/Discngine/fpocket.git")
+    url = "https://github.com/Discngine/fpocket/archive/refs/tags/4.1.tar.gz"
+
+    version(
+        "4.1",
+        "1a2af2d3f2df42de67301996db3b93c7eaff0375f866443c0468dcf4b1750688",
+        extension="tar.gz",
```

Comment:
```suggestion
```

Since the extension here matches that in the URL you shouldn't need to explicitly call it out in the version.